docs/assembly.rst

```diff
@@ -291,10 +291,10 @@ In the grammar, opcodes are represented as pre-defined identifiers.
 |                         |     |   | 8 byte value, ``this`` is the current contract's address        |
 |                         |     |   | as a 20 byte value and ``s`` is a big-endian 256-bit value      |
 +-------------------------+-----+---+-----------------------------------------------------------------+
-| call(g, a, v, in,       |     | F | call contract at address a with input mem[in...(in+insize))     |
```

Comment:
I don't think we should call these "function parameters", because we do not call a function. I would leave it as it is.